File: Problems/Solved/Easy/2716.ts
function minimizedStringLength(s: string): number {
    let result = 0;
    let caches: string[] = [];
    for (let i = 0; i < s.length; i++) {
        if (!caches.includes(s[i])) {
            result++;
            caches.push(s[i]);
        }
    }
    return result;
}
const s: string = "baadccab";

console.log(minimizedStringLength(s));

File: Problems/2716.rs
fn main() {}
#[cfg(test)]
mod tests {
    use super::*;

    #[test]
    fn test_case() {
        let s = "baadccab".to_string();
        let expected_output = 4;

        assert_eq!(minimized_string_length(s), expected_output);
    }
}

pub fn minimized_string_length(s: String) -> i32 {
    let mut result = 0;
    let mut caches: Vec<char> = Vec::new();
    for c in s.chars() {
        if !caches.contains(&c) {
            result += 1;
            caches.push(c);
        }
    }
    result
}
